TombEngine/Game/effects/effects.h

```diff
@@ -206,7 +208,7 @@ void TriggerCyborgSpark(int x, int y, int z, short xv, short yv, short zv);
 void TriggerExplosionSparks(int x, int y, int z, int extraTrig, int dynamic, int uw, int roomNumber);
 void TriggerExplosionSmokeEnd(int x, int y, int z, int uw);
 void TriggerExplosionSmoke(int x, int y, int z, int uw);
-void TriggerFireFlame(int x, int y, int z, FlameType type);
+void TriggerFireFlame(int x, int y, int z, FlameType type, Vector3 color1 = Vector3(1 ,1 ,1 ), Vector3 color2 = Vector3(1, 1, 1));
```

Comment:
Please correct spacing here (comma should go directly after number and not other way around)

Reply:
fixed